docs/en/siem/siem-ui.asciidoc

```diff
@@ -65,29 +65,164 @@ versions, machine type, and so forth.
 [[network-ui]]
 === Network
 
-The Network view provides key network activity metrics, facilitates
-investigation time enrichment, and provides network event tables that enable
-interaction with the Timeline. You can drag and drop items of interest from the
-Network view to Timeline for further investigation.
+The Network view provides key network activity metrics and an interactive map,
+facilitates investigation time enrichment, and provides network event tables that
+enable interaction with the Timeline. You can drag and drop items of interest from
+the Network view to Timeline for further investigation.
 
 [role="screenshot"]
 image::network-ui.png[]
```

Comment:
May need one more iteration on this screenshot as the maps styling has been updated from red/green to blue/magenta for source and destination points.

Reply:
Thanks - updated the screenshot